app/controllers/concerns/idv_step_concern.rb

```diff
@@ -41,6 +41,10 @@ def flow_path
   private
 
   def confirm_ssn_step_complete
+    if IdentityConfig.store.in_person_ssn_info_controller_enabled
+      # mark ssn step as complete for FSM
```

Comment:
should the comment read for non-fsm rather than fsm?

Reply:
@svalexander  Although we are in the non-FSM flow- I am actually marking the FSM step as complete. Here is my thinking (let me know your thoughts please)...

When the users manually changes the url to the FSM ssn step (because they did not visit it, it is not getting marked as complete) so they are able to revisit it (not expected behavior) rather than being redirected back to Verify (expected behavior). If we mark it as complete- they will be redirect to Verify Info- and will have to get to SSN by using the Update button. 

This will need to get removed in your delete PR.

Reply:
ahhh i see what you're saying, ok the comment makes sense then